collect_app/src/main/java/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -669,4 +674,20 @@ public MediaPlayer getPlayer() {
     public int getPlayColor() {
         return playColor;
     }
+
+    protected void startActivityForResult(Intent intent, int requestCode, @StringRes int errorRes) {
```

Comment:
I think it would be better not to use a common Android method name here because it makes it unclear that it's a custom method. I'm not sure exactly how to describe this. `requestActivityResultOrShowErrorToast`? It's a bit long but thorough. It's surprising that a possible side effect is a toast being shown. There should definitely be a comment here. `startActivityForResultOrShowErrorToast`?

Reply:
:+1: